lib/LazyLoadingGhostFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager;
+
+use ProxyManager\Configuration;
+use ProxyManager\Factory\LazyLoadingGhostFactory as BaseLazyLoadingGhostFactory;
+
+/**
+ * Extends the LazyLoadingGhostFactory of the ProxyManager, to make it configureable according to the needs of Sulu.
+ */
+class LazyLoadingGhostFactory extends BaseLazyLoadingGhostFactory
+{
+    public function __construct($cacheDirectory)
+    {
+        if (!is_dir($cacheDirectory)) {
+            mkdir($cacheDirectory);
```

Comment:
I have checked it, the permissions are fine: 
![image](https://cloud.githubusercontent.com/assets/405874/9849251/505b493a-5ae9-11e5-84da-c3893736e230.png)